schema/mongoose.ts

```diff
@@ -789,6 +789,75 @@ export const Video = new Schema<types.Video>(
   { collection: 'Video', timestamps: { createdAt: 'createdDate', updatedAt: 'updatedDate' } }
 );
 
+export const VideoParticipant = new Schema<types.VideoParticipant>(
+  {
+    name: { type: String, required: true },
+    profileId:  { type: mongoose.Schema.Types.ObjectId, ref: 'Profile', required: true },
```

Comment:
not required, because we wont always have a profile for every participant in the system

Reply:
okay would change the contrainst on that